app/controllers/user/books_controller.rb

```diff
@@ -4,4 +4,25 @@ class User::BooksController < ApplicationController
   def index
     @userbooks = UserBook.where(user_id: current_user.id)
   end
+
+  def new; end
+
+  def create
+    # if current_user.nil? bounce to 404 page
+    
+    book = BookFacade.find_by_isbn(params[:isbn])
+    if book.isbn.nil?
+      flash[:failure] = "Oops! That book cannot be found."
+      redirect_to new_user_book_path and return
+    end
+
+    user_book = UserBook.new(status: 'available', user_id: current_user.id, isbn: book.isbn)
```

Comment:
Future work: have the user confirm to add this book to user shelf. Right now it just adds the book, even if user accidentally inserted wrong ISBN